src/domains/citizen-auth-common/03_storage.tf

```diff
@@ -277,3 +277,25 @@ resource "azurerm_storage_queue" "profiles_to_sanitize" {
   name                 = "profiles-to-sanitize"
   storage_account_name = module.io_citizen_auth_storage.name
 }
+
+
+module "azure_storage_account" {
+  source = "github.com/pagopa/dx//infra/modules/azure_storage_account?ref=main"
+
+  environment                          = local.itn_environment
+  resource_group_name                  = data.azurerm_resource_group.data_rg_itn.name
+  tier                                 = "l"
+  subnet_pep_id                        = data.azurerm_subnet.subnet_pep_itn.id
+  private_dns_zone_resource_group_name = "${local.prefix}-${local.env_short}-rg-common"
+
+  subservices_enabled = {
+    blob  = false
+    file  = false
+    queue = true
+    table = true
+  }
+
+  force_public_network_access_enabled = false
+
+  tags = var.tags
+}
```

Comment:
As you can see [here](https://github.com/pagopa/io-infra/blob/72b981ad2fdbfbfea569a9343c57c167e133c7b4/src/domains/citizen-auth-common/03_storage.tf#L269-L279), the original storage account has a table and a queue defined via terraform.
Can you please add them for the new storage account as well?

```
resource "azurerm_storage_table" "profile_emails_itn" {
  name                 = "profileEmails"
  storage_account_name = module.io_citizen_auth_storage_itn.name
}

resource "azurerm_storage_queue" "profiles_to_sanitize_itn" {
  name                 = "profiles-to-sanitize"
  storage_account_name = module.io_citizen_auth_storage_itn.name
}
```